src/ansys/fluent/core/utils/file_transfer_service.py

```diff
@@ -240,20 +262,7 @@ def download(
 
     def exit(self):
         """Stop the container."""
-        id_ports_info = subprocess.run(
-            [f"docker ps"], shell=True, stdout=subprocess.PIPE
-        )
-        id_ports = str(id_ports_info).split("\\n")
-        active_container_ids = [
-            id_port.split(" ")[0]
-            for id_port in id_ports
-            if str(self.host_port) in id_port
-        ]
-        for container_id in active_container_ids:
-            try:
-                subprocess.Popen(f"docker kill {container_id}", shell=True)
-            except Exception:
-                pass
+        self.server.kill()
```

Comment:
I believe we should use `stop`  instead, as it already has a built-in configurable timeout of 10 seconds, after which it will send a kill signal (see [docs](https://docker-py.readthedocs.io/en/stable/containers.html#docker.models.containers.Container.stop)).

This together with the other change I suggested above would mean:
```suggestion
        self.container.stop()
```

Reply:
Done. Thank you.